<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bar3DPlotAreaDock</class>
 <widget class="QWidget" name="Bar3DPlotAreaDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>813</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="TimedLineEdit" name="leName"/>
       </item>
       <item row="1" column="2">
        <widget class="ResizableTextEdit" name="teComment"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lComment">
         <property name="text">
          <string>Comment:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lName">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="chkVisible">
         <property name="text">
          <string>Visible</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lXColumn">
         <property name="text">
          <string>Data:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QFrame" name="frameColumns">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMesh">
      <attribute name="title">
       <string>Mesh</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="9" column="0">
        <widget class="QLabel" name="lTheme">
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="cbShadowQuality"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lXRot">
         <property name="text">
          <string>X Rotation</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSlider" name="slYRot">
         <property name="maximum">
          <number>90</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="slZoom">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lColor">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="KComboBox" name="cbTheme"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lYRot">
         <property name="text">
          <string>Y Rotation</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Shadow Quality:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="KColorButton" name="kcbColor"/>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QSlider" name="slXRot">
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lZoom">
         <property name="text">
          <string>Zoom</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lColumn">
         <property name="text">
          <string>Column</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="cbNumber"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>TimedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>src/kdefrontend/widgets/TimedLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ResizableTextEdit</class>
   <extends>QTextEdit</extends>
   <header>src/kdefrontend/widgets/ResizableTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>chkVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
